docs/infra/set-up-database.md

```diff
@@ -26,6 +26,29 @@ make infra-configure-app-database APP_NAME=<APP_NAME> ENVIRONMENT=<ENVIRONMENT>
 `APP_NAME` needs to be the name of the application folder within the `infra` folder. By default, this is `app`.
 `ENVIRONMENT` needs to be the name of the environment you are creating. This will create a file called `<ENVIRONMENT>.s3.tfbackend` in the `infra/app/service` module directory.
 
+### (Optional) Enable any database extensions that require `rds_superuser`
+
+To enable some extensions, such as [pgvector](https://github.com/pgvector/pgvector), requires the `rds_superuser` role. You can enable any such extensions via the `superuser_extensions` configuration variable, and set them to either enabled or disabled.
+
+For example, to enable the pgvector extension:
+
+```terraform
+# infra/app/app-config/env-config/outputs.tf
+
+output "database_config" {
+  value = var.has_database ? {
+    ...
+    superuser_extensions = {
+      "vector": true,
+    }
+  } : null
+}
+```
+
+If you're not sure whether you need any of these extensions, you can skip this step and come back to it later.
+
+Note that this should only be used for extensions that require the `rds_superuser` role to be created. For most extensions, you should instead enable them as part of your application's standard database migrations.
```

Comment:
Is there an easy way to see which extensions require the `rds_superuser` role to enable?

Reply:
Any extensions that aren't on this list: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.Extensions.Trusted -- I will add a reference to this!

Reply:
Thank you!